docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+---
+title: How to Stake on EON
+---
+
+# How to Stake on EON
+A forger is required to run a node. See [How to Locally Build and Deploy](../04-develop_and_deploy_smart_contracts/04-local_build_and_deploy.md) for information on creating and running a node.  
+The following steps require some basic general knowledge; a web application for staking is under construction and will be made available soon.
+
+#### Create Forger Keys
+On your local node, create new forger keys by making a `POST` request to `/wallet/createVrfSecret`.
+The request will return your `publicKey` (later called `vrfPubKey`), while the privateKey will be stored in the node's wallet.
+
+```
+{
+  "result" : {
+    "proposition" : {
+      "publicKey" : "4439cbfd50af1b846e5ef06889d3192ef7a459bdd4640dc6da506062de43113c80"
+    }
+  }
+}
+``` 
+
+At this point there are two ways of staking on EON: via HTTP endpoint or via smart contract.
+#### HTTP Endpoint
+Make a `POST` request to `/transaction/makeForgerStake` with body
+```
+{
+    "forgerStakeInfo": {
+        "ownerAddress": "e8674062598b86d8258257a4d79e99d15594d864", 
+        "blockSignPublicKey": "e8674062598b86d8258257a4d79e99d15594d864", 
```

Comment:
Shouldn't the blockSignPublicKey be an Edwards curve key pair, not a secp256k1 account key pair? (generated with `/wallet​/createPrivateKey25519` or `sctool generatekey`)

Reply:
Sorry, I should have been more clear in my comment. It's not just the wrong format for blockSignPublicKey, but in the "Create Forger Keys" section a step to generate a blockSign key pair should also be added.